SourcetrailExtension/Wizard/WindowCreateCDB.cs

```diff
@@ -181,6 +181,43 @@ private CreationResult CreateCdb()
 			return result;
 		}
 
+		private void CreateCompilationCommands(Utility.QueuedFileWriter fileWriter, EnvDTE.Project project, ref int projectsProcessed, bool lastProject = false)
```

Comment:
New helper function to write a project's compile commands. Takes an optional bool indicating whether this project is the last one.